docs/user-guide/index.md

```diff
@@ -57,6 +57,16 @@ the different pieces:
 * [Connectors](#connectors)
 * [Formatters](#formatters)
 
+## Permission Issue
+
+If you receive an `EACCES` error when installing `sonar`, the recommended
+solution is to [change npm's default directory](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npms-default-directory-to-another-directory) and then try again. So far, one such permission issue has been reported
+when user tries to install `sonar` on [Windows Subsystem for Linux](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide). Dependency
+`canvas-prebuilt` throws an `EACCES` error during the installation process,
+and this [issue](https://github.com/sonarwhal/sonar/issues/308) was resolved
+adopting the recommended solution. You can find detailed steps on how to change
+the npm default directory [here](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npms-default-directory-to-another-directory).
```

Comment:
I was also capable of reproducing this with Mac.
I had node installed via the installer in their website but it looks like if you use [Homebrew](https://brew.sh/) that problem doesn't show up (didn't had the time to test).
@alrra how did you installed node in your machine?